astropy/table/tests/test_item_access.py

```diff
@@ -242,13 +242,13 @@ def test_np_where(self, table_data):
         assert len(t2) == 0
         assert isinstance(t2, table_data.Table)
 
-    def test_np_integers(self, table_data):
+    def test_np_integers(self, rng, table_data):
         """
         Select rows using numpy integers.  This is a regression test for a
         py 3.3 failure mode
         """
         t = table_data.Table(table_data.COLS)
-        idxs = np.random.randint(len(t), size=2)
+        idxs = rng.integers(len(t), size=2)
         t[idxs[1]]
```

Comment:
The pull request that added the test says that in Python 2 `isinstance(np.int64(0), int)` was `True`, but in Python 3 it is `False`. This caused a bug with Python 3 where trying to access rows in a table using a `np.integer` didn't work. From the pull request description it is clear that the test is about the type of `idxs` and not its value. Generating `idxs` randomly is therefore not any better than hardcoding a value.

I should also point out that this test insists on generating two random values despite using only one (and needing zero). And the real problem with this test is that it doesn't assert anything. It just ensures that `t[idxs[1]]` doesn't raise an error, but it doesn't check if the output is a `Row` and not a `Column` or a `Table` or `None`.

Reply:
You're right. I'll patch this one to remove rng entierly.